provider/data/decimal/fingerprints.csv

```diff
@@ -1,12 +1,12 @@
 decimal/symbols@1, <lookup>, 1316B, 252 identifiers
-decimal/symbols@1, <total>, 9698B, 2191B, 49 unique payloads
-decimal/symbols@1, af, 197B, 34B, 8f773f51e85a65c1
-decimal/symbols@1, ar, 202B, 39B, 29e2dc764329c56
-decimal/symbols@1, ar-BH, 202B, 49B, c3f15eb63fa35608
+decimal/symbols@1, <total>, 4798B, 2926B, 49 unique payloads
```

Comment:
Suggestion: baked gets a lot smaller, but postcard gets a bit bigger. I think we should use Index8 on the MultiFieldsVarULE

Reply:
We can't do that with the struct yet, but we can do that with VarTuple!

Reply:
That's a compelling reason to use VarTuple.

I want make_var to take an optional IndexFoo parameter though (we have an issue for it), so I might do that instead.

Reply:
Is the default Index32 or Index16 in `make_varule`?

Based on the size diff, I can only see how Index32 would account for such a large percentage change per payload (e.g. 39B to 54B).

Reply:
I think it defaults to Index32 out of caution. If we make it configurable I think it should default to 16 and we can override.

Reply:
I think we should do that, so I'll do that work regardless of the choice of VarTuple or whatever